python-avd/tox.ini

```diff
@@ -20,6 +20,7 @@ deps =
     pytest
     PyYAML>=6.0.0
     pydantic>=2.3.0
+    referencing>=0.35.0
```

Comment:
so adding referencing but not json_schema?

Reply:
these are extra dependency and depending on the version of json-schema this is needed to add referencing.